trap/src/main/java/com/cursorinsight/trap/datasource/gesture/internal/TrapWindowCallback.kt

```diff
@@ -53,7 +55,15 @@ class TrapWindowCallback internal constructor(
     }
 
     override fun dispatchTouchEvent(event: MotionEvent?): Boolean {
-        touchHandlers.forEach { it(event) }
+        try {
+            touchHandlers.forEach { it(event) }
+        } catch (ex: Exception) {
+            Log.e(
```

Comment:
What if there are hundreds of thousands of failiing events?  I recommend adding a failsafe to keep sending failing log messages after some time.

Reply:
Please check and resolve the same issue below in TrapAccelerometerCollector, TrapGravityCollector, etc.

Reply:
I've changed the logging